File: cpp/reverse_number.cpp
#include <iostream>
using namespace std;

int main() {
    int num, reversed = 0;
    cout << "Enter an integer: ";
    cin >> num;

    while (num != 0) {
        int remainder = num % 10;
        reversed = reversed * 10 + remainder;
        num /= 10;
    }

    cout << "Reversed number = " << reversed << endl;
    return 0;
}

File: javascript/reverse_number.js
let num = parseInt(prompt("Enter an integer: "));
let reversed = 0;

while (num != 0) {
    let remainder = num % 10;
    reversed = reversed * 10 + remainder;
    num = Math.floor(num / 10);
}

console.log("Reversed number =", reversed);
